andela/codango
# #128843401 Create Team Page

#### What does this PR do?
It creates a team page for Codango development team members.

#### Description of Task to be completed?
A react team component was created for the team page, and new team members were added to the team page.

#### How should this be manually tested?
- Visit the review app [here](https://codango-staging-pr-152.herokuapp.com)
- Click on the "new" navigation menu
- Check out the team page on the new homepage

#### Screenshots
![screen shot 2016-09-06 at 4 48 07 pm](https://cloud.githubusercontent.com/assets/17270426/18280682/cb2aa926-7451-11e6-9c36-4aca1c96f29a.png)



#### What are the relevant pivotal tracker stories?
#128843401 - Create our Team Page
